src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,56 @@
+package duke;
+
+public class Ui {
+
+    public Ui() {
+
+    }
+
+    public void welcome() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        System.out.println("What can I do for you?");
+    }
+
+    public void showLine() {
+        System.out.println("_________________________________");
+    }
+
+    public void showError(DukeException error) {
+        System.out.println(error);
+    }
+
+    public void bye() {
+        System.out.println("Bye! Hope to see you again soon!");
+    }
+
+    public void list(TaskList tasklist) {
```

Comment:
I believe the coding standards suggest that method names should be verbs. In this case, 'list' could be a noun. Maybe a better option would be to name the method 'listAll'.